homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -161,15 +168,13 @@
 
 # Air Fresh
 ATTR_CO2 = "co2"
+NAME_AIRFRESH = "Air Fresh"
 
 # Map attributes to properties of the state object
 AVAILABLE_ATTRIBUTES_AIRPURIFIER_COMMON = {
     ATTR_TEMPERATURE: "temperature",
     ATTR_HUMIDITY: "humidity",
```

Comment:
Temperature and humidity should also be separate sensor entities. Please move these as well.

Reply:
This Air Purifier model (3C) does not have neither temperature nor humidity

Reply:
@MartinHjelmare this comment seems unrelated to the pull request, which simply aims to add support for the 3C model.

The temperature and humidity values are already listed as attributes in the original home-assistant repo: https://github.com/home-assistant/core/blob/dev/homeassistant/components/xiaomi_miio/fan.py#L167

Could you open a separate issue to aim to improve this by splitting out the mentioned attributes as sensor entities?

Reply:
The current code is already scoped beyond adding support for the 3C model, which you can see if you read the sensor platform additions. If we only want that scope, we'll need to slim that part down.